WTWSMS/common/religions/00_religions.txt

```diff
@@ -426,12 +431,13 @@ christian = {
 		religious_clothing_priest = 5
 
 		character_modifier = { ### If Antinomians can sin without consequence, few other Christians will like them and they'll sin a ton. ###
+			same_religion_opinion = 50
 			fertility = 0.3
 			sex_appeal_opinion = 5
-			christian_opinion = -15
+			christian_opinion = -50

```

Comment:
It is not too brutal. It's totally historical. Antinomians were incestors: it was a crime that was universally hated by everyone.

No Christian would ever get along with an Antinomian. Even Seducers hated incest. 

As for incestors being fine with Antinomians, I don't know how to code that. "ANTINOMIAN_OPINION" wouldn't work because it's the sinner's opinion of Antinomians, and not the other way around.